plugins/error.rst

```diff
@@ -1,4 +1,32 @@
 Error Plugin
 ============
 
-TODO: explain the error plugin
+Error plugin will transform response with specific status code into exception:
+
+ * 400-499 status code are transformed into `Http\Client\Plugin\Exception\ClientErrorException`;
+ * 500-599 status code are transformed into `Http\Client\Plugin\Exception\ServerErrorException`
```

Comment:
at least in the symfony doc, such lists have a `;` at the end except for the last line which has `.` - ok if we do the same here? so add a dot after this line?